frontend/src/scenes/experiments/ExperimentResult.tsx

```diff
@@ -67,6 +67,14 @@ export function ExperimentResult(): JSX.Element {
                                 </Col>
                             ))}
                         </div>
+                        <div className="flex justify-between py-2 border-t">
```

Comment:
small nit for Line 60: Can you update it to say metric when appropriate as well, like here: https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/experiments/ExperimentResult.tsx#L107

Reply:
eh, I'm not sure I understand, can you double check? 👇 (assuming you meant line 70)

Reply:
Sorry for not being clear, I meant line 60. Just calling it Exposure here is fine, but when using experiment goals like 'avg per user', calling the number as "count" doesn't make sense. We do this right for < 4 variants, but it still says "Count" always for > 4 variants